tests/require.test.js

```diff
@@ -124,16 +125,6 @@ describe('Plugin for require blocks', () => {
     `);
   });
 
-  it('ignores non-function factories', () => {
```

Comment:
Hmm... why is this test no longer valid?

Reply:
Restored